bundles/org.openhab.core.transform/src/test/java/org/openhab/core/transform/FileTransformationProviderTest.java

```diff
@@ -137,8 +139,8 @@ public void testMissingExtensionIsIgnored() throws IOException {
         Path extensionMissing = Path.of("extensionMissing");
```

Comment:
I think you can use `Path extensionMissing = transformationPath.resolve("extensionMissing")` and remove `path` completely. (also in other places)

Reply:
good catch! I've adjusted it. There was only one other instance like this right below I think.